### ISSUE TITLE
Example of label transfer with saturn? 

### ISSUE THREAD

**Issue by dariotommasini** [snap-stanford/SATURN#58]
Hi @Yanay1 , 

Is there an example in one of the vignettes for how to do label transfer using SATURN? 

Thanks!
Dario

**Comment by Yanay1** [comment]
To do label transfer there are a variety of off the shelf methods you can use. One that we use for scoring is to use [sklearn logistic regression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) classifier to transfer labels from one species to the other. 

In that case, the features (X) would be the SATURN embeddings and the labels (Y) would be the cell types. The training data would be the embeddings from one species, and you would apply it to the embeddings from the other. 

Other approaches like using clustering or nearest centroid could also work.
